lib/Fhp/Syntax/Bins.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I'd be in favor of renaming this to `GebuchteCamtUmsaetze` in order to match the specification more closely. If you then move it into the `CAZ` directory, this also allows reverting the change in `BaseDescriptor.php`.

Reply:
I see `GebuchteCamtUmsaetze.php` now. But I also still see this `Bins.php`. Should it be deleted? (Or has it been deleted already, but GitHub continues to show it for some reason?)

Reply:
you are right this can be deleted